### PR TITLE
Added node 6.

### PR DESCRIPTION
**Pull request by arb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,18 +12,18 @@
...
-    "markdown-toc": "0.11.x"
+    "lab": "10.x.x",
+    "markdown-toc": "0.12.x"
   },
   "scripts": {
     "test": "lab -a code -t 100 -L",
     "test-cov-html": "lab -a code -r html -o coverage.html -L",
     "toc": "node generate-toc.js",
-    "version": "npm run toc && git add README.md"
+    "postversion": "npm run toc && git add README.md"
```
Comment (Marsup) [inline package.json]: That works ? Isn't it happening a bit late ?

Comment (arb) [inline package.json]: It actually never worked with "version" so I thought I'd try it a different way. This way, `version` has finished and i just amend it into that version commit. Seems like it worked.

Comment (Marsup) [inline package.json]: I've been using it for a long while on joi, always worked.

Comment (arb) [inline package.json]: Don't know what to tell ya... it literally NEVER worked for me. So I switched it to something else. It always complained about not being able to add README.md.

**Pull request merged**